packages/host/app/components/operator-mode/container.gts

```diff
@@ -153,6 +168,113 @@ export default class OperatorModeContainer extends Component<Signature> {
     return get(card, path.join('.'));
   }
 
+  @action
+  onSelectedCards(selectedCards: Card[], stackItem: StackItem) {
+    let selected = cardSelections.get(stackItem);
+    if (!selected) {
+      selected = new TrackedArray([]);
+      cardSelections.set(stackItem, selected);
+    }
+    selected.splice(0, selected.length, ...selectedCards);
+  }
+
+  get selectedCards() {
+    return this.operatorModeStateService
+      .topMostStackItems()
+      .map((i) => cardSelections.get(i) ?? []);
+  }
+
+  get copyState(): CopyButtonState | undefined {
```

Comment:
I think this is probably complex enough logic that the copy button should probably be its own component